tck/features/TemporalArithmeticAcceptance.feature

```diff
@@ -0,0 +1,193 @@
+#
+# Copyright (c) 2015-2018 "Neo Technology,"
+# Network Engine for Objects in Lund AB [http://neotechnology.com]
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Attribution Notice under the terms of the Apache License 2.0
+#
+# This work was created by the collective efforts of the openCypher community.
+# Without limiting the terms of Section 6, any Derivative Work that is not
+# approved by the public consensus process of the openCypher Implementers Group
+# should not be described as “Cypher” (and Cypher® is a registered trademark of
+# Neo4j Inc.) or as "openCypher". Extensions by implementers or prototypes or
+# proposals for change that have been documented or implemented should only be
+# described as "implementation extensions to Cypher" or as "proposed changes to
+# Cypher that are not yet approved by the openCypher community".
+#
+
+#encoding: utf-8
+
+Feature: TemporalArithmeticAcceptance
+
+  Scenario: Should add or subtract duration to or from date
+    Given an empty graph
+    And having executed:
+      """
+      CREATE (:Val {prop: date({year:1984, month:10, day:11})}),
+             (:Durations {prop: [duration({years: 12, months: 5, days: 14, hours: 16, minutes: 12, seconds: 70, nanoseconds: 2}),
+              duration({months:1, days: -14, hours: 16, minutes: -12, seconds: 70}),
+              duration({years: 12.5, months: 5.5, days: 14.5, hours: 16.5, minutes: 12.5, seconds: 70.5, nanoseconds: 3})] })
```

Comment:
Creating multiple nodes is difficult because of the order in which they can be matched. And I do want to have a defined order on the result table, i.e. `the result should be, in order`. I will go for a Scenario Outline

Reply:
That's all right. As an alternative, we could also do `ORDER BY`.